scripts/openroad/or_droute.tcl

```diff
@@ -24,7 +24,14 @@ if {[catch {read_def $::env(CURRENT_DEF)} errmsg]} {
 
 set_thread_count $::env(ROUTING_CORES)
 
-detailed_route -param $::env(tritonRoute_tmp_file_tag).param
+detailed_route\
+    -guide $::env(CURRENT_GUIDE)\
+    -output_guide $::env(TRITONROUTE_FILE_PREFIX).guide\
+    -output_maze $::env(TRITONROUTE_FILE_PREFIX)_maze.log\
+    -output_drc $::env(TRITONROUTE_RPT_PREFIX).drc\
+    -droute_end_iter $::env(ROUTING_OPT_ITERS)\
+    -or_seed 42\
```

Comment:
Why are you using or_seed?

Reply:
Fixing the seed gives the test a lot more replicability. I’m not sure if you fix a seed by default and I don’t feel like dedicating time testing that.

Reply:
The seed is fixed - this is for trying to see if alternative values give a better result.

Reply:
Eh, well, this particular seed has come up with our current benchmarks so unless you’re planning to deprecate this option I will not rock the boat unnecessarily.